opentsdb/tsdb.go

```diff
@@ -2,6 +2,7 @@
 package opentsdb // import "bosun.org/opentsdb"
 
 import (
+	"bosun.org/cache"
```

Comment:
ok that's an interesting design. so basically it's up to the thing that calls opentsdb/graphite queries to have their own cache before invoking queries.  But there can be multiple things invoking tsdb/graphite.  I'll give this a shot and see how I can make this work.